spec/plurimath/latex/metanorma_exampes_spec.rb

```diff
@@ -0,0 +1,199 @@
+require_relative "../../../lib/plurimath/math"
+
+RSpec.describe Plurimath::Latex::Parser do
```

Comment:
@suleman-uzair for all imported specs/tests, can you label the source of the formula? e.g. `metanorma/mn-samples-iso:{path name}`?

Reply:
Thanks, I've also updated it.